### DISCUSSION TITLE
QField with Bad Elf Flex Mini (iOS and Windows) accuracy

### DISCUSSION THREAD

**Discussion by kellymcgee** [opengisch/QField#5718] [Q&A]
Has anyone been able to use the Bad Elf Flex Mini GPS (Standard) with QField and get better than the 5.0-meter horizontal accuracy reported in QField?  The Bad Elf Flex app shows > 2.0-meter horizontal accuracy.

**Comment by Meykaefer** [comment]
Hi @kellymcgee,

This question is very similar to #4577.

I am not 100% sure about this, but as far as I understand, an iOS limitation prevents QField from receiving additional information from your Bad Elf Flex Mini receiver.

To explain: I think your GPS provides a position with 2m accuracy to the iPad/iPhone, but QField itself cannot read the accuracy data, i.e. 2m, and will display 5m instead, which is based or defined somewhere in the location services of the iOS. I have observed the same thing while working with my Trimble Catalyst at 1cm accuracy. You can test if your device properly uses the GPS by placing it at a defined spot, stepping away from it, and walking around it in a circle. Since your device remains stationary, the displayed GPS position should also remain stationary and should not follow you while you are walking a circle. Of course, the point might shift around based on the GPS device's accuracy (i.e. 2m), but if you walk a circle with a 6m radius, it should be easy to observe if the Bluetooth connection is stable and if the accuracy is 2m.

If you are using a Bluetooth connection, it seems there is no easy way for iOS devices to get a better horizontal accuracy reported. Maybe you can get the proper accuracy reported if you connect your GPS via UDP or TCP? Take a look at m-kuhn's reply in #4577.
